blog/2021-09-21-intro-to-isolated-vm.md

```diff
@@ -0,0 +1,163 @@
+---
+tags:
+  - temporal
+  - community
+posted_on_: 2021-09-14T00:00:00Z
+slug: intro-to-isolated-vm
+title: Introduction to isolated-vm in Node.js
+author: Valeri Karpov
+author_title: Community Member
+author_image_url: https://avatars.githubusercontent.com/u/1620265?v=4
+release_version: V1.9.1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!--truncate-->
+
+## Introduction
+
+[Temporal's Node SDK](/docs/node/introduction) makes heavy use of the [isolated-vm npm package](https://www.npmjs.com/package/isolated-vm) to ensure your [Workflows are deterministic](/docs/node/determinism).
```

Comment:
Right now we do but that's an implementation detail.
Maybe instead of talking about isolated-vm, talk about v8 isolates and say that we use the isolated-vm package for controlling isolates in nodejs.